core/src/main/java/jenkins/util/SystemProperties.java

```diff
@@ -199,7 +199,7 @@ public static String getString(String key) {
         }
         
         if (LOGGER.isLoggable(Level.CONFIG)) {
-            LOGGER.log(Level.CONFIG, "Property (not found): {0} => {1}", new Object[] {key, value});
+            LOGGER.log(Level.CONFIG, "Property (not found): {0} => {1}", new Object[] {key, null});
```

Comment:
I think it's better to change the message, it makes no sense to print something we know that won't have a value.

Reply:
You are right. I changed it.